manual/tools/caml_tex2.ml

```diff
@@ -12,17 +11,18 @@ let camlbunderline = "\\<"
 let camleunderline = "\\>"
 
 let start newline out s args =
-  Printf.fprintf out "%s%s" camlbegin s;
-  List.iter (Printf.fprintf out "{%s}") args;
-  if newline then Printf.fprintf out "\n"
+  Format.fprintf out "%s%s" camlbegin s;
+  List.iter (Format.fprintf out "{%s}") args;
+  if newline then Format.fprintf out "\n"
```

Comment:
I wondered for a moment whether `newline` should be interpreted as `@\n` or `@.`, but to by honest I don't really understand the differences here and I think that the current code is probably fine -- as long as it works...

Reply:
The printing is basically bypassing the Format's layout engine since we are exactly mirroring the input, thus there is no pratical difference now. ( I made the switch to format to avoid mixing Format and Printf since the compiler's printing function were using Format).